mod/groups/views/default/resources/elements/groups/alpha.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Sure, I had that there, but then thought it would be closer to resources. But, yes, I agree.
As a general rule, I want to move all elgg_list_entities() and elgg_view_form() calls into their own views, so that we don't have to touch resource views unnecessarily.